.github/workflows/test-build.yml

```diff
@@ -26,15 +27,15 @@ jobs:
           go-version: ${{ matrix.go-version }}
       - name: Display Go Version
         run: go version
-      - name: mod-check
+      - name: Run Go mod check
         run: make mod-check
-      - name: lint
+      - name: Run lint
         run: make lint
-      - name: test
+      - name: Run test
         run: make test
-      - name: test-benchmarks
+      - name: Run test benchmarks
         run: make test-benchmarks
-      - name: check-protos
+      - name: Run check protos and generate Go code from the proto files
```

Comment:
[nit] We could drop the "run" from each of these names, for example:

* Check go.mod
* Lint
* Test
* Test benchmarks
* Check protos